contracts-periphery/Makefile

```diff
@@ -0,0 +1,14 @@
+# include .env file and export its env vars
+# (-include to ignore error if it does not exist)
+-include .env
+forkBlock = 10465092
+
+all:; clean install test
+# Clean the repo
+clean  :; forge clean
+# Install the Modules
+install :; forge install
+# Test your contract on a forked network
+test   :; forge test -f ${ETH_RPC_URL} --fork-block-number ${forkBlock}
```

Comment:
Running `make test` I currently get:

```
The application panicked (crashed).
Message:  couldn't deploy:
   0: deployment failed: Revert

Location:
   /Users/runner/work/foundry/foundry/evm/src/executor/mod.rs:448

Backtrace omitted. Run with RUST_BACKTRACE=1 environment variable to display it.
Run with RUST_BACKTRACE=full to include source snippets.
Location: /Users/runner/work/foundry/foundry/forge/src/runner.rs:226

Backtrace omitted. Run with RUST_BACKTRACE=1 environment variable to display it.
Run with RUST_BACKTRACE=full to include source snippets.
make: *** [test] Abort trap: 6
```

Reply:
Not sure why this is, it's working for me. Have you tried `foundryup`?

Reply:
Foundry currently has poor support for showing useful error messages when your test contract deployments fail in the constructor. It panics instead of showing a trace, and that's what's happening here.

The most likely reason is you're forking from the wrong network. Even though there's no explicit constructor, the test contract calls `token.balanceOf(umbra)` in the top storage portion, and that's executed during construction. Since you are on the wrong network, that token will not exist and that call will revert, causing deployment to fail. Two things we should do:
1. Document what to fork against, OR remove the need for forking, as we are discussing in a comment above
2. The test contract should not make any calls that set values in a constructor: variables should all be set in the `setUp` function to prevent opaque errors like this

Reply:
> The most likely reason is you're forking from the wrong network. 

My RPC url is for Rinkeby so I don't think that's it

Reply:
RIP my whole theory haha. We're removing the need to fork anyway so probably not worth debugging